core/backend.py

```diff
@@ -96,6 +96,9 @@ def classifier_pre_process(self, embeddings):
         Returns:
             embeddings = numpy array of shape (1,10,128), dtype=float32.
         """
+        if 0 < time_stamp < embeddings.shape[0]:
+            embeddings = embeddings[time_stamp:time_stamp+10,:]
```

Comment:
Maybe you should add a warning here if the `if` statement is false

Reply:
Yeah, I just didn't know how to surface it properly atm, but that was my best idea too